src/main/groovy/life/qbic/datamodel/dtos/business/Offer.groovy

```diff
@@ -145,18 +191,23 @@ class Offer {
             return this
         }
 
-        Builder vat(double vat) {
-            this.vat = vat
+        Builder itemsWithOverhead(List<ProductItem> itemsWithOverhead) {
+            this.itemsWithOverhead= itemsWithOverhead
             return this
         }
 
-        Builder net(double net) {
-            this.net = net
+        Builder itemsWithoutOverhead(List<ProductItem> itemsWithoutOverhead) {
+            this.itemsWithoutOverhead= itemsWithoutOverhead
             return this
         }
```

Comment:
These methods have been removed -> breaking changes. Please add them back and deprecate them.

Reply:
But there is no variable like net to be returned. This will give an error